CHANGES.rst

```diff
@@ -150,6 +150,7 @@ Testing
 
 * pytest: report the result of ``test_run_epubcheck`` as ``skipped`` instead of
   ``success`` when Java and/or the ``epubcheck.jar`` code are not available.
+* utils: make HTTP(S) test servers use unused ports by default.
```

Comment:
It's not really "by default" since you are using port *0*. Instead, could you perhaps add a default port number to the server class which can *also* be specified when creating the context manager?